src/task.jl

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
maybe add it to the jvm.jl. It gets difficult to read IMO if code is spread out into very small files.

Reply:
I consolidated the new functions code in jvm.jl. I added the `JULIA_COPY_STACKS` global to JavaCall.jl and returned the evaluation to `__init__` which was necessary anyways.